tests/test_exceptions.py

```diff
@@ -50,9 +50,26 @@
 
 def test_lambda_fw_exception() -> None:
     exp = LambdaFWException(message="Nope")
+    assert str(exp) == "[500] Nope"
     assert exp.message == "Nope"
     assert exp.status_code == 500
-    assert isinstance(exp.get_response(request_id=""), Response)
+
+    response = exp.get_response(request_id="test-req-id")
+    assert isinstance(response, Response)
```

Comment:
I don't think this assertion is needed - with `mypy` and the below line, it will be pretty hard to make it wrong 😉